tracing-undertow-jakarta/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.palantir.external-publish-jar'
+apply plugin: 'com.palantir.revapi'
+
+dependencies {
+    api project(':tracing')
+    api 'io.undertow:undertow-core:2.3.0.Alpha1'
```

Comment:
I thought only undertow-servlet differed in 2.3.0 (jakarta by default) where undertow-core shouldn't be impacted.

Reply:
Oh maybe, let me look through it.

Reply:
Yup, you were right purged.